recipes/quantms-rescoring/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "quantms-rescoring" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/quantms_rescoring-{{ version }}.tar.gz
+  sha256: 25143539cf4789666d90922314b84bac12e097e10a59b520698e7361b8b1a504
+
+build:
+  entry_points:
+    - rescoring=quantmsrescore.rescoring:main
+  noarch: python
+  run_exports:
+    - {{ pin_subpackage('quantms-rescoring', max_pin="x.x") }}
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-cache-dir -vvv"
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<3.11
+    - pip
+    - poetry-core
+  run:
+    - python >=3.8,<3.11
+    - click
+    - pyopenms >=2.4.0
+    - ms2rescore ==3.0.3
+    - psm-utils ==0.8.3
+    - pandas
+    - numpy
+    - deepLC ==2.2.38
+    - scipy ==1.13.1
+    - pygam
+    - protobuf >=3.9.2 
+test:
+  imports:
+    - quantmsrescore
+  commands:
+    - rescoring --help
+    - rescoring ms2rescore --help
+  requires:
+    - pip
+
+about:
+  home: https://www.github.com/bigbio/quantms-rescoring
+  summary: PSM rescoring python package with scripts and helpers for the quantms workflow
+  license: MIT
```

Comment:
Seems to be apache: https://github.com/bigbio/quantms-rescoring/blob/main/LICENSE